### ISSUE TITLE
Type error: cannot unpack non-iterable DataParallel objects

### ISSUE THREAD

**Issue by JiaqiWww** [aleju/LossAccPlotter#5]
I use the code like below and meet two problem,first with this one in laplotter.py line 255 and I comment it.
#plt.figure(self.fig.number)
After it I meet this problem:
Type error: cannot unpack non-iterable DataParallel objects
How can I get rid of it?

def main():
	plotter = LossAccPlotter()
	
	for epoch in range(start_epoch, start_epoch+50):
			scheduler.step()
			train(epoch)
			test(epoch)
			loss_train, acc_train = net.train()
			loss_val, acc_val = net.validate()
			plotter.add_values(epoch, loss_train=loss_train, acc_train=acc_train,loss_val=loss_val, acc_val=acc_val)
			plotter.block()
